code/modules/projectiles/projectile/bullets.dm

```diff
@@ -112,6 +112,20 @@
 /obj/item/projectile/bullet/midbullet3
 	damage = 30
 
+/obj/item/projectile/bullet/midbullet3/hp
+	damage = 45
+	armour_penetration = -70

```

Comment:
I'd personally recommend a value of something like 40 and -40 that way these aren't 100% worthless against someone with a teensy amount of armor, but aren't super dead kill against those without it.